lib/bundler.js

```diff
@@ -94,7 +94,7 @@ function bundleInstructions(instructions, bundler) {
   // Write the instructions to a file in our node_modules folder.
   // The location should not really matter as long as it is .gitignore-ed
   var instructionsFile = path.resolve(__dirname,
-    '..', 'node_modules', 'instructions.json');
+    '..', 'instructions.json');
```

Comment:
I am little bit worried about adding a generated file that looks like it is a part of loopback-boot sources. Can we use a different name that would make it clear that the file is a temporary artefact created during the build? Few suggestions: `.generated-instructions.js` or `.browser.instructions.js`.